C++/src/Driver.cpp

```diff
@@ -1,20 +1,29 @@
 #include <PCH.h>
 
 #include "cryptography/AES.h"
+#include "cryptography/QuantumPermutationPad.h"
 
-
-int main() {
-
-    // std::cout << "Hello World" << std::endl;
-
+void AES_test() {
     QPP::AES aes;
-
     std::string input = "Hello Wrorld";
     std::string key = "abcdefghijklmnop";
     std::cin >> input;
     std::cin >> key;
     std::string result = aes.encrypt(input, key, 128);
     std::cout << result << std::endl;
+}
+int main() {
+
+    std::cout << "Hello World" << std::endl;
+
+    // AES_test();
+
+    std::vector<int> seed = {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16};
+    QPP::QuantumPermutationPad qpp(seed);

```

Comment:
Create using pointer via QPP....* a = new QPP... . Exceed stack limit here that's why there is a seg fault

Reply:
resolved

Reply:
Still have seg falut